kata06/app/controllers/employees_controller.rb

```diff
@@ -0,0 +1,35 @@
+class EmployeesController < ApplicationController
+    def index
+        @employees = Employee.all
+    end
+    def new
+        @employee = Employee.new
+    end
+
+    def create
+        data = params.require(:employee).permit(:name, :email, :posicion, :num_empleado, :private_num)
+        Employee.create(data)
+
+        redirect_to employees_path
+    end
+
+    def edit
+        @employee = Employee.find(params[:id])
+    end
+
+    def update
+        @employee = Employee.find(params[:id])
+        data = params.require(:employee).permit(:name, :email, :posicion, :num_empleado, :private_num)
```

Comment:
You can create a private method called employee_params like this:

```ruby
def employee_params
  params.require(:employee).permit(:name, :email, :posicion. :num_empleado, :private_num)
end
```

Reply:
i will edit that for my next PR!